tools/travis/install_byond.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+source dependencies.sh
```

Comment:
`source buildByond.conf` should work instead of this.
might have to move spacemandmm out and up here